sdk/ml/azure-ai-ml/azure/ai/ml/_restclient/dataset_dataplane/models/_azure_machine_learning_workspaces_enums.py

```diff
@@ -7,16 +7,15 @@
 # --------------------------------------------------------------------------
 
 from enum import Enum
-from six import with_metaclass
 from azure.core import CaseInsensitiveEnumMeta
 
 
-class DataflowType(with_metaclass(CaseInsensitiveEnumMeta, str, Enum)):
+class DataflowType(str, Enum, metaclass=CaseInsensitiveEnumMeta):
```

Comment:
These enums are potentially generated, so unless the team updates their autorest version, these changes might be lost.